content/en/network_performance_monitoring/installation.md

```diff
@@ -122,14 +127,51 @@ If you need to use Network Performance Monitoring on other systems with SELinux
 
 If these utilities do not exist in your distribution, follow the same procedure but using the utilities provided by your distribution instead.
 
+### Windows systems
+
+Data collection for Windows systems is now available in public beta for versions 2016 or later. Note that NPM currently monitors Windows hosts only, and not Windows containers. DNS metric collection is not currently supported for Windows systems.
```

Comment:
```suggestion
Data collection for Windows systems is available in public beta for versions 2016+. **Note**: NPM monitors Windows hosts only, and not Windows containers. DNS metric collection is not supported for Windows systems.
```

Reply:
The latest version of Windows is 2004 (the naming doesn't make sense to me, either), so "2016 or later" is intentional copy to avoid the confusion of "2016+" Will make the other edits!